### PR TITLE
Eplanning adapter: Get domain from page

### PR DESCRIPTION
**Pull request by chino117**

### CODE REVIEW

File: adapters/eplanning/eplanning.go [hunk 1]
```diff
@@ -114,6 +114,13 @@ func (adapter *EPlanningAdapter) MakeRequests(request *openrtb.BidRequest, reqIn
 	var pageDomain string
 	if request.Site != nil && request.Site.Domain != "" {
 		pageDomain = request.Site.Domain
+	} else if request.Site != nil && request.Site.Page != "" {
+		u, err := url.Parse(request.Site.Page)
+		if err != nil {
+			errors = append(errors, err)
```
Comment (SyntaxNode) [inline adapters/eplanning/eplanning.go:120]: Can you please add a test for this error handler?

File: adapters/eplanning/eplanning.go [hunk 2]
```diff
@@ -114,6 +114,13 @@ func (adapter *EPlanningAdapter) MakeRequests(request *openrtb.BidRequest, reqIn
 	var pageDomain string
 	if request.Site != nil && request.Site.Domain != "" {
 		pageDomain = request.Site.Domain
+	} else if request.Site != nil && request.Site.Page != "" {
```
Comment (SyntaxNode) [inline adapters/eplanning/eplanning.go:117]: Nitpick: The `request.Site != nil` check is repeated. What do you think of pushing down the `Site.Domain` and `Site.Page` check within a single nil check?

**Comment by chino117** [comment]
Perfect, no problem

**Comment by chino117** [comment]
The test is cleared as the error response from url.parse differs between versions 1.13 and 1.14.2 and there is no way that works in both versions. You could leave this error handler untested but if you don't prefer it, can you think of a practical way to test it?

**Comment by SyntaxNode** [comment]
> The test is cleared as the error response from url.parse differs between versions 1.13 and 1.14.2 and there is no way that works in both versions. You could leave this error handler untested but if you don't prefer it, can you think of a practical way to test it?

Ah. yes. It's an unfortunate side effect of relying on framework generated messages. We first encountered this when moving to Go 1.13 when some of the error messages changed on how they included the name of the part of the struct with a marshal error. 

We've provided a way to test these kinds of error messages using a simple regex. Most adapter json tests use the literal comparison mode, but there's also an option to use regex which we introduced to address this exact problem.

Here's an example:
https://github.com/prebid/prebid-server/blob/cce496720f13d30d45154ca7d80e81884fb9a1ac/adapters/beintoo/beintootest/supplemental/invalid-response-unmarshall-error.json#L57

**Pull request merged**